patches/chrome-installer-linux-debian-additional_deps.patch

```diff
@@ -0,0 +1,11 @@
+diff --git a/chrome/installer/linux/debian/additional_deps b/chrome/installer/linux/debian/additional_deps
+index 7646e729a5044..92bca46102841 100644
+--- a/chrome/installer/linux/debian/additional_deps
++++ b/chrome/installer/linux/debian/additional_deps
+@@ -31,3 +31,6 @@ libvulkan1
+ 
+ # For OS integration.
+ xdg-utils (>= 1.0.2)
++
```

Comment:
do not add empty lines or extra lines for comments in patches https://github.com/brave/brave-browser/wiki/Patching-Chromium

Reply:
also merge_package_versions.py in merge_deb_dependencies can take multiple input arguments so it would be better to add `args += brave_deb_dependencies` to the end of `merge_deb_dependencies`

Reply:
I think it's reasonable to just subsume the new dep under the existing "OS integration" heading. If you want to comment, consider "Optional use of GeoClue2 for Location Services" to reflect that it's a weak dependency.